framework/web_app/src/services/files/ImageService.ts

```diff
@@ -0,0 +1,36 @@
+export default class ImageService {
+    // encodeToBase64 = (imageFile: File): Promise<string> =>
+    //     new Promise((resolve, reject) => {
```

Comment:
we may want to include this commented out function if required for saving the image files

Reply:
wait no, i remember now...was never needed because the offer's original image is never touched, so you don't need to change it back

sooooo.... we could keep this encode function if we think we'd need it for some reason? hard to say but currently there's no use for it

plus we can delete the example below

Reply:
Inclined to leave comments & examples for now because saving images or retrieving saved images is not working atm, that's why I had that ignore.

Will address img's when doing the my products page.

Reply:
Resolve if happy with this